hs_model_program/views.py

```diff
@@ -37,10 +37,10 @@ def get_model_metadata(request):
                 software_language=mpmeta.modelProgramLanguage,
                 operating_sys=mpmeta.modelOperatingSystem,
                 url = protocol+"://"+request.get_host()+"/resource/"+resource_id+"/",
-                modelEngine = mpmeta.modelEngine.split(';'),
-                modelSoftware=mpmeta.modelSoftware.split(';'),
-                modelDocumentation=mpmeta.modelDocumentation.split(';'),
-                modelReleaseNotes=mpmeta.modelReleaseNotes.split(';'),
+                modelEngine = mpmeta.get_engine_list(),
```

Comment:
@jsadler2 I am not seeing for example where in the javascript the context variable ```modelEngine``` is being used.